src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -428,6 +432,16 @@ class MySimulation : public SimulationHelper<simDim>
         ForEach<VectorSpeciesWithIonizer, particles::CallIonization<bmpl::_1>, MakeIdentifier<bmpl::_1> > particleIonization;
         particleIonization(forward(particleStorage), cellDescription, currentStep);
 
+        /* SynchrotronPhotons */
+        typedef typename PMacc::particles::traits::FilterByFlag<VectorAllSpecies,
+                                                                synchrotronPhotons<> >::type AllSynchrotronPhotonsSpecies;
```

Comment:
This are all radiating particles (electrons) but **no photons**. Please rename this list to avoid confusion.